packages/react-storage/src/components/StorageBrowser/actions/handlers/__tests__/listLocations.spec.ts

```diff
@@ -1,7 +1,117 @@
-import { listLocationsHandler } from '../listLocations';
+import {
+  listCallerAccessGrants,
+  ListLocationsOutput,
+  LocationCredentialsProvider,
+} from '../../../storage-internal';
+import {
+  listLocationsHandler,
+  ListLocationsHandlerInput,
+} from '../listLocations';
+import { LocationAccess } from '../types';
+import { parseLocations } from '../utils';
+
+jest.mock('../../../storage-internal');
+
+const mockListCallerAccessGrants = jest.mocked(listCallerAccessGrants);
+
+const generateMockLocations = (size: number, mockLocations: LocationAccess) =>
+  Array<LocationAccess>(size).fill(mockLocations);
 
 describe('listLocationsHandler', () => {
-  it('is null', () => {
-    expect(listLocationsHandler).toBeNull();
+  const accountId = 'account-id';
+  // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Interesting, is eslint not like an async function declared, even though it isn't called?

Reply:
yes.

Reply:
Updated to remove eslint disable